index.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=0.5">
```

Comment:
I played a little bit with the numbers and I think I have found a perfect ratio. So now on most of the mobile devices it is much better than it was before you have added this viewport!!! Look, for example, how it looks on iPhone now:
![image](https://user-images.githubusercontent.com/26466644/32352477-a6fa4bca-c021-11e7-8c7b-a809fb0e9e74.png)

Thank you, Kolyan! You have improved the project a lot!!! 🥇

Reply:
`initial-scale=1.0` means, that page will be loaded without any scaling.
`initial-scale=0.5` means, that page will be loaded with some scale and has a width of the current viewport.
It's like you open a page in the browser and then zoom out the scale.

I'm not sure, that this is a good idea.
If you open a page on the real mobile browser, you will see that some parts of the page have a bad quality, especially images, some borders disappeared. Of course, it depends on device screen resolution. Maybe, on iPhone it will look good :)

You can read more about viewport meta tag [here](http://html5.by/blog/viewport-meta-tag/).
This tag is useful for mobile browsers and in most cases makes the page easy to use.
But for charts, phones don't have enough space in portrait orientation.